src/modules/cluster/cluster-list/cluster-list.js

```diff
@@ -0,0 +1,84 @@
+(function() {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.controller("clusterController", clusterController);
+
+    /*@ngInject*/
+    function clusterController($scope, $state, $interval, config, $filter, utils) {
+        var vm = this,
+            key,
+            len,
+            temp = [],
+            clusterData,
+            cluster,
+            i;
+
+        vm.importCluster = importCluster;
+        vm.getOrderByArg = getOrderByArg;
+        vm.displayArg = displayArg;
+
+        init();
+
+        function init() {
+            _createClusterList();
+        }
+
+        /*Refreshing list after each 30 second interval*/
+        var timer = $interval(function () {
+            
+            utils.getClusterData()
+                .then(function(data) {
+                    config.clusterData = data;
+                });
+            
+            init();
+
+        }, 1000 * config.initData.refreshIntervalTime );
+
+        /*Cancelling interval when scope is destroy*/
+        $scope.$on('$destroy', function() {
+            $interval.cancel(timer);
+        });
+
+        function displayArg() {
+            console.log(vm.orderByArg);
+        }
+
+        function importCluster() {
+            $state.go("import-cluster");
+        }
+
+        function getOrderByArg() {
+            return vm.orderByArg + "";
+        }
+
+        function _createClusterList() {
+            clusterData = config.clusterData.clusters;
+            len = clusterData.length;
+            temp = [];
+
+            for ( i = 0; i < len; i++) {
+                cluster = {};
+                key = Object.keys(clusterData[i]);
+                
+                cluster.name = clusterData[i][key[0]].tendrl_context.sds_name;
+                cluster.id = clusterData[i][key[0]].tendrl_context.cluster_id;
+                cluster.alertCnt = clusterData[i].stats.alert_cnt;
+                cluster.storage = clusterData[i].stats.storage;
+                cluster.storage.percentUsed = clusterData[i].stats.storage["percent-used"];
```

Comment:
@gnehapk yes i understand .. but only one assignment for storage is enough
i means :   
cluster.storage = clusterData[i].stats.storage; (enough )
cluster.storage.percentUsed = clusterData[i].stats.storage["percent-used"]; (this is not required 
because already in first line you have assigned this value in cluster.storage than why again you are specifically assigning the object properties .

if you are thinning for "percent-used" will not cause any issue . than i can help you .. actually json data can contain "hyphen" .. and also we can handle this in html itself.